<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>74</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>404</width>
    <height>74</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>404</width>
    <height>74</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VSCode Desktop generator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="workspace_file_path">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>code-workspace's Path</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Directory">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Directory</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="Generate_name">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Desktop Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Generate">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAdd_New_Template">
   <property name="text">
    <string>Add New Template</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
